### PR TITLE
Add Oppo

### LINKED ISSUE #6197: Request: OPPO
**Brand Name:** OPPO
**Website:** https://www.oppo.com
**Alexa rank:** 7,149
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
oppo

**Issue:** closes #6197

**Similarweb rank:** 6,101. See https://www.similarweb.com/website/oppo.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from brand website header https://www.oppo.com
Color: `#2d683d` Taken from https://www.figma.com/community/file/832815970641696814/OPPO-Media-Kit

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9487,6 +9487,12 @@
             "source": "https://opnsense.org/about/legal-notices/",
             "guidelines": "https://opnsense.org/about/legal-notices/"
         },
+        {
+            "title": "Oppo",
```
Comment (LitoMore) [inline _data/simple-icons.json:9491]:
```suggestion
            "title": "OPPO",
```

See this: https://www.oppo.com/en/about/.

They call themself `OPPO`.

Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Fixed

**Comment by MlgmXyysd** [comment]
seems it updated to black color lol
image

**Comment by PratyushJayachandran** [comment]
> seems it updated to black color lol image

We will go with their Figma resource link: https://www.figma.com/community/file/832815970641696814/OPPO-Media-Kit

**Comment by MlgmXyysd** [comment]
&gt; Last updated 3 years ago




---Original---
From: ***@***.***&gt;
Date: Sat, Oct 7, 2023 00:17 AM
To: ***@***.***&gt;;
Cc: "Jaida ***@***.******@***.***&gt;;
Subject: Re: [simple-icons/simple-icons] Add Oppo (PR #9556)




  
seems it updated to black color lol 
  
We will go with their Figma resource link: https://www.figma.com/community/file/832815970641696814/OPPO-Media-Kit

**Comment by PratyushJayachandran** [comment]
> &gt; Last updated 3 years ago

Yes but it is official brand resource. We should not make up our own assumptions when they provide a resource. We can update the logo anytime a change happens though, like here: #9542

**Review by adamrusted — APPROVED** [review]

**Pull request merged**